Dockerfile

```diff
@@ -21,12 +21,13 @@
 # limitations under the License.
 #
 
-FROM python:3.6 
+FROM python:3.6-slim-buster 
 
 # Requirements as per https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html
 RUN apt-get update \
- && apt-get install -y bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev \
- && rm -rf /var/lib/apt/lists/*
+ && apt-get install -y bzip2 xz-utils zlib1g libxml2-dev libxslt1-dev libgomp1 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man/?? /usr/share/man/??_*
```

Comment:
A single layer is always best, but do you known if this would this remove any of the license or copyright notices?

Reply:
Not sure if, worth checking

Reply:
I checked and you were right: some apt packages can contain copyright notices information there.
I removed this locations from Dockerfile: and, surprise, it does not affect the final image size – still 558MB)